github-git/3-fork-repo.md

```diff
@@ -0,0 +1,100 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.16.4
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+(fork-repository)=
+# How to Fork a GitHub Repository 
+
+
+:::{admonition} What you will learn:
+:class: tip
+
+In this lesson, you will learn how to <kbd>fork</kbd> (or create a copy of) a GitHub repo into your own **GitHub.com** account. You can practice forking the pyOpenSci example repository. 
+::: 
+
+:::{admonition} Activity: Fork a repository and modify a file
+
+**1. Fork the pyOpenSci practice GitHub repository**
+*******
+
+Fork the [pyOpenSci example repo](https://github.com/pyOpenSci/pyos-demo-package-contribute). Remember that a fork is a copy of a repository that is owned by someone else or an organization that lives in your GitHub account.
```

Comment:
```suggestion
Fork the [pyOpenSci example repo](https://github.com/pyOpenSci/pyos-demo-package-contribute). Remember that a fork is a copy of a repository that is owned by someone else.
```

A mild suggestion, but I don't think a reader wanting to learn the basics of github is going to manage multiple namespaces under one GitHub account and trade PRs between them. This seem to distract from the (slightly simple) definition.

Reply:
in training my lab on git (most are electrical engineers not software ppl), another point of confusion is "what's the difference between a fork and a branch," not sure if that's worth expanding on here bc not sure what level we're pitching this lesson at